google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/testing/LocalPubSubHelperTest.java

```diff
@@ -50,25 +46,10 @@ public void testCreate() {
   }
 
   @Test
-  public void testOptions() {
-    LocalPubSubHelper helper = LocalPubSubHelper.create();
-    PubSubOptions options = helper.getOptions();
-    assertTrue(options.getProjectId().startsWith(PROJECT_ID_PREFIX));
-    assertTrue(options.getHost().startsWith("localhost:"));
-    assertSame(NoCredentials.getInstance(), options.getCredentials());
-  }
-
-  @Test
   public void testStartStopReset() throws IOException, InterruptedException, TimeoutException {
     LocalPubSubHelper helper = LocalPubSubHelper.create();
```

Comment:
I guess let's keep it for now - it's not conceptually related to the existence of the handwritten layer.